lib/ProductOpener/Display.pm

```diff
@@ -10329,9 +10329,10 @@ sub display_structured_response($)
 		# remove the languages field which has keys like "en:english"
 		# keys with the : character break the XML export
 
-		delete $request_ref->{structured_response}{product}{languages};
-		delete $request_ref->{structured_response}{product}{category_properties};
-		delete $request_ref->{structured_response}{product}{categories_properties};
+		my @foo = ("languages", "category_properties", "categories_properties");
```

Comment:
Instead create a function that we can call like this:

delete_fields($product_ref, ["languages", "category_properties", "categories_properties"]);

Reply:
Done! Looks much righter now.

Reply:
Thanks for the feedback! just made the changes